docs/topics/kotlin-ide.md

```diff
@@ -15,6 +15,22 @@ Kotlin plugin is bundled with each IntelliJ IDEA release.
 
 Read more about IntelliJ IDEA in the [official documentation](https://www.jetbrains.com/help/idea/discover-intellij-idea.html).
 
+## Fleet
+
+> JetBrains Fleet is currently in Public Preview, and at this stage is free to use.
+{type="note"}
+
+[JetBrains Fleet](https://www.jetbrains.com/fleet/) is a polyglot IDE and code editor that provides advanced support for Kotlin
+and a streamlined experience for Kotlin developers. You can use Fleet as a code editor to quickly make targeted edits,
+or turn on Smart Mode and transform it into a full-fledged IDE with code intelligence features.
+
+On top of that, with Fleet, you can quickly open and run Kotlin Multiplatform projects targeting the Android, iOS, and desktop
+platforms (Fleet's Smart Mode selects the appropriate code-processing engine). The IDE supports testing and debugging your code
+on all the platforms your project targets. You can also navigate between Kotlin Multiplatform code and code written in other
+languages interoperable with Kotlin.
+
+See the [Use Fleet for Multiplatform development](https://www.jetbrains.com/help/kotlin-multiplatform-dev/fleet.html) tutorial to get started.
```

Comment:
Android Studio and IDEA sections have info that Kotlin plugin is bundled with each release. 
Should we mention how it is working in the Fleet?

Reply:
Yes, we should. I was confused: plugins are not explicitly supported, but plugin architecture is already in place.